tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/Test_org_eclipse_swt_internal_SVGRasterizer.java

```diff
@@ -0,0 +1,107 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Vector Informatik GmbH and others.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse
+ * Public License 2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Michael Bangas (Vector Informatik GmbH) - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.swt.tests.junit;
+
+import static org.eclipse.swt.tests.junit.SwtTestUtil.assertSWTProblem;
+import static org.junit.Assert.fail;
+
+import java.io.File;
+import java.net.URL;
+
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.SWTException;
+import org.eclipse.swt.graphics.Image;
+import org.eclipse.swt.graphics.ImageData;
+import org.eclipse.swt.graphics.ImageDataProvider;
+import org.eclipse.swt.graphics.ImageFileNameProvider;
+import org.eclipse.swt.widgets.Display;
+import org.junit.Before;
+import org.junit.Test;
+
+public class Test_org_eclipse_swt_internal_SVGRasterizer {
+
+	Display display;
+
+	ImageFileNameProvider imageFileNameProvider = zoom -> {
+		String fileName = "collapseall.svg";
+		return getPath(fileName);
+	};
+
+	ImageDataProvider imageDataProvider = zoom -> {
+		String fileName = "collapseall.svg";
+		return new ImageData(getPath(fileName));
+	};
+
+	@Before
+	public void setUp() {
+		display = Display.getDefault();
```

Comment:
You could also inline this instead of storing it in a instance field, or at least retrieve it via a `getDisplay()` method on demand instead of storing it in a field.